probe-rs/targets/STM32U0_Series.yaml

```diff
@@ -0,0 +1,1651 @@
+    range:
+      start: 0x20000000
+      end: 0x20002000
+    cores:
+    - main
+  - !Nvm
+    name: Flash
+    range:
+      start: 0x8000000
+      end: 0x8010000
+    is_boot_memory: true
+    cores:
+    - main
+  - !Generic
+    name: SRAM2
+    range:
+      start: 0x20002000
+      end: 0x20003000
+    cores:
+    - main
+  flash_algorithms:
+  - stm32u0xx_64k
+- name: STM32U031R6Ix
+  cores:
+  - name: main
+    type: armv6m
+    core_access_options: !Arm
+      ap: 0
+      psel: 0
+  memory_map:
+  - !Ram
+    name: SRAM1
+    range:
+      start: 0x20000000
+      end: 0x20002000
+    cores:
+    - main
+  - !Nvm
+    name: Flash
+    range:
+      start: 0x8000000
+      end: 0x8008000
+    is_boot_memory: true
+    cores:
+    - main
+  - !Generic
+    name: SRAM2
+    range:
+      start: 0x20002000
+      end: 0x20003000
+    cores:
+    - main
+  flash_algorithms:
+  - stm32u0xx_32k
+- name: STM32U031R6Tx
+  cores:
+  - name: main
+    type: armv6m
+    core_access_options: !Arm
+      ap: 0
+      psel: 0
+  memory_map:
+  - !Ram
+    name: SRAM1
+    range:
+      start: 0x20000000
+      end: 0x20002000
+    cores:
+    - main
+  - !Nvm
+    name: Flash
+    range:
+      start: 0x8000000
+      end: 0x8008000
+    is_boot_memory: true
+    cores:
+    - main
+  - !Generic
+    name: SRAM2
+    range:
+      start: 0x20002000
+      end: 0x20003000
+    cores:
+    - main
+  flash_algorithms:
+  - stm32u0xx_32k
+- name: STM32U031R8Ix
+  cores:
+  - name: main
+    type: armv6m
+    core_access_options: !Arm
+      ap: 0
+      psel: 0
+  memory_map:
+  - !Ram
+    name: SRAM1
+    range:
+      start: 0x20000000
+      end: 0x20002000
+    cores:
+    - main
+  - !Nvm
+    name: Flash
+    range:
+      start: 0x8000000
+      end: 0x8010000
+    is_boot_memory: true
+    cores:
+    - main
+  - !Generic
+    name: SRAM2
+    range:
+      start: 0x20002000
+      end: 0x20003000
+    cores:
+    - main
+  flash_algorithms:
+  - stm32u0xx_64k
+- name: STM32U031R8Tx
+  cores:
+  - name: main
+    type: armv6m
+    core_access_options: !Arm
+      ap: 0
+      psel: 0
+  memory_map:
+  - !Ram
+    name: SRAM1
+    range:
+      start: 0x20000000
+      end: 0x20002000
+    cores:
+    - main
+  - !Nvm
+    name: Flash
+    range:
+      start: 0x8000000
+      end: 0x8010000
+    is_boot_memory: true
+    cores:
+    - main
+  - !Generic
+    name: SRAM2
+    range:
+      start: 0x20002000
+      end: 0x20003000
+    cores:
+    - main
+  flash_algorithms:
+  - stm32u0xx_64k
+- name: STM32U073C8Tx
+  cores:
+  - name: main
+    type: armv6m
+    core_access_options: !Arm
+      ap: 0
+      psel: 0
+  memory_map:
+  - !Ram
+    name: SRAM1
+    range:
+      start: 0x20000000
+      end: 0x20008000
+    cores:
+    - main
+  - !Nvm
+    name: Flash
+    range:
+      start: 0x8000000
+      end: 0x8010000
+    is_boot_memory: true
+    cores:
+    - main
+  - !Generic
+    name: SRAM2
+    range:
+      start: 0x20002000
```

Comment:
SRAM2 is incorrect for all U073s and U083s here. Datasheets say

> SRAM2: 8 Kbytes with hardware parity check, located at address 0x1000 0000
SRAM2 is also mapped at address 0x2000 8000